std/typecons.d

```diff
@@ -7645,6 +7645,12 @@ public:
     {
         return opBinary!op(flag);
     }
+
+    bool opDispatch(string name)() const if (__traits(hasMember, E, name))
+    {
+        enum e = __traits(getMember, E, name);
+        return (mValue & e) == e;
```

Comment:
Why this instead of `cast(bool)(mValue & e)`?

Reply:
```d
auto mValue = 2;
auto e = 3;

(mValue & e).writeln;
((mValue & e) == e).writeln;
```

https://run.dlang.io/is/l1LXwy

Reply:
Ah, ok, I was thinking with the wrong intuition for what this syntax meant.

Reply:
A good optimizer should be able to lower this to identical code for single bit flags, but in particular multi-bit flags are tedius to test.
Good old `flags & Enum.combined` can be used when you want to test for any instead of all flags.
Unfortunately the docs for BitFlags are in bad shape, so I just added this to the all-explaining example.